modules/serverless-installing-service-mesh-control-plane.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+= Installing the ServiceMeshControlPlane
+
+After you install the ServiceMesh operator, you can install the control plane. The following procedure installs a version of ServiceMesh that acts as an ingress to your applications.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference.
+====
+
+[source,yaml]
+.Sample servicemesh-installation.yaml
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+metadata:
+ name: basic-install
+ namespace: istio-system <1>
+spec:
+ istio:
+   global:
+     multitenant: true
+     proxy:
+       autoInject: disabled
+     omitSidecarInjectorConfigMap: true
+     disablePolicyChecks: false
+   istio_cni:
+     enabled: true
+   gateways:
+     istio-ingressgateway:
+       autoscaleEnabled: false
+       type: LoadBalancer
+     istio-egressgateway:
+       enabled: false
+     cluster-local-gateway:
+       autoscaleEnabled: false <2>
+       enabled: true
+       labels:
+         app: cluster-local-gateway
+         istio: cluster-local-gateway
+       ports:
+         - name: status-port
+           port: 15020
+         - name: http2
+           port: 80
+           targetPort: 80
+         - name: https
+           port: 443
+   mixer:
+     enabled: false
+     policy:
+       enabled: false
+     telemetry:
+       enabled: falsely 
+   pilot:
+     autoscaleEnabled: false
+     sidecar: false
+   kiali:
+     enabled: false
+   tracing:
+     enabled: false
+   prometheus:
+     enabled: false
+   grafana:
+     enabled: false
+   sidecarInjectorWebhook:
+     enabled: false
+----
+<1> Required. The installation program prompts you for this value.
+<2> Autoscaling is disabled in this version, as this is currently not intended for production use.
```

Comment:
Maybe slightly reword this to sound more like: This is a minimum version to kick the tires. Please adjust these values to your production needs as necessary?